woo-razorpay.php

```diff
@@ -1758,7 +1823,11 @@ public function updateOrder(& $order, $success, $errorMessage, $razorpayPaymentI
                 {
                     $wcOrderId = $order->get_id();
 
-                    $is1ccOrder = get_post_meta( $wcOrderId, 'is_magic_checkout_order', true );
+                    if (OrderUtil::custom_orders_table_usage_is_enabled()) {
```

Comment:
Instead of checking this flag in every function and file, can we declare it globally once and reuse this?